ftest.sh

```diff
@@ -145,9 +145,16 @@ if [ \"\${HOSTNAME%%%%.*}\" != \"${nodes[0]}\" ]; then
         fi
     fi
 
+    tmpfs_size=16777216
```

Comment:
Perhaps.  I think I'm undecided at this point.  I'm not really sure why, but something is just not sitting right about it.  But since I don't know why (at this point) I could just as easily be wrong.  I think it's something about the fact that the tmpfs is a substitute for real storage, only really useful for testing.

In any case, this feels like it's something for a different PR.